### ISSUE TITLE
Fastmail-FolderBadgeControls not working upon log in, but only after restart *when* logged in..

### ISSUE THREAD

**Issue by MelchiorGaspar** [michaelstepner/fastmail-FolderBadgeControls#2]
so yeah.. maybe it has something to do with Firefox v52 and GreaseMonkey... not sure but I thought I would open a new ticket here so we can look into this together..

I noticed that your Fastmail-FolderBadgeControls script was not working when I would login to the new/current FastMail UI  [https://www.fastmail.com/login/](url)

only when preforming a restart after have already been logged in ... did the script start working...

Strange?! lol

ps: I edited two things in the script as well... I changed the line endings from LF (Linux) to CR+LF (Windows)
, do you run Linux?

and changed 
`// @include      https://www.fastmail.com/mail/*`
to
`// @match       https://www.fastmail.com/mail/*`

Hoping that would help too, but these of these two changes didn't do anything as far as I can tell... :/

**Comment by michaelstepner** [comment]
Not sure why you had to restart after logging in, but I'm glad it worked after you restarted.

I expect the script will be correctly parsed regardless of whether it uses Windows or Mac/Linux line endings.  Not sure if `@include` vs `@match` would change anything, but it seems it didn't based on your experience.

If you have any patches to contribute that improve how this Greasemonkey script works on your computer, feel free to reopen or submit a new pull request!

**Comment by MelchiorGaspar** [comment]
ok, maybe I should open a support ticket with GreaseMonkey then if its not on your end...

**Comment by michaelstepner** [comment]
I imagine the problem is with the **Fastmail-FolderBadgeControls** script. I just don't know what the problem is, and am not actively developing or supporting this software at the moment.

It's working for me, and available in case it's useful for others. But I can't promise any updates or debugging.

**Comment by MelchiorGaspar** [comment]
np, I'm writing a issue ticket for GreaseMonkey now...

**Comment by MelchiorGaspar** [comment]
Also I checked the Browser Console... and I didn't see anything related to either GreaseMonkey or your script before and after a restart...

EDIT:
I will download an archive copy of Firefox version 51.0.1 and test it out with this script to see if it is indeed an issue caused by FF v52.0.0 and GreaseMonkey..


nope the issue occurred in FF v51.0.1 as well... :/
